test/celeritas/optical/OpticalMockTestBase.cc

```diff
@@ -186,6 +188,15 @@ void OpticalMockTestBase::build_import_data(ImportData& data) const
         data.optical_materials[0].rayleigh.scale_factor = 1;
         data.optical_materials[0].rayleigh.compressibility
             = native_value_from(Compressibility{7.658e-23});
+        data.optical_materials[0].wls.mean_num_photons = 2;
+        data.optical_materials[0].wls.time_constant
+            = native_value_from(TimeSecond(1e-9));
+        // Reemitted photon energy range (visible light)
+        data.optical_materials[0].wls.component.x
+            = {1.65e-6, 2e-6, 2.4e-6, 2.8e-6, 3.26e-6};
```

Comment:
I feel a pang of regret for not using eV for the optical tracking loop 😅 (not like it matters in most cases). These are in MeV I guess? And does the spectrum below integrate to 1 over the energy range?

Reply:
Yep, in MeV :). We use the spectrum to calculate the CDF, so it does get normalized during setup time.